internal/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -1,74 +1,15 @@
 ---
 id: an-introduction-to-iota
-title: An Introduction to IOTA
-description: A basic introduction to the IOTA DLT Technology. The use-cases and
-  the Ecosystem developed around it.
+title: IOTA
+description: A basic introduction to the IOTA protocol.
 ---
 
-# An Introduction to IOTA
+# IOTA
 
-**This topic gives a brief overview and describes some of the main differences between IOTA's Tangle and a blockchain.**
+IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any middlemen or middlebanks.
 
-Blockchains and the Tangle both fall under the same category of distributed ledger technology (DLT).
+As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.
```

Comment:
```suggestion
As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if the same entity controlled all nodes, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node will join the network, so any DLT needs a way to protect itself. IOTA approaches this differently than protocols.
```

Reply:
One suggestion from my side, remove the mention of " a DLT maintains a SINGLE ledger. Many DLTs handle multiple ledgers and sidechains / parachains. IOTA will do the same, so at one point it will become misleading. A DLT can manage multiple networks, of which each handles a single ledger of records of token ownership

Reply:
And we have to have a strategy how to address the new nomenklatur. We decided to name everything blocks now.... How is this handled here? Messages should be completely removed from the wiki